.circleci/config.yml

```diff
@@ -185,7 +186,7 @@ jobs:
             curl --location https://github.com/stedolan/jq/releases/download/jq-1.6/jq-osx-amd64 --output $HOME/.bin/jq
             chmod +x $HOME/.bin/jq
       - go_build:
-          cmd: "smartdeps"
+          cmd: "deps"
```

Comment:
what's the reason for s/smartdeps/deps/g ? I didn't see any reference to this change in a commit message - more curious than anything.

Reply:
`smartdeps` don't exist anymore. Default `deps` manages deps smartly.

Reply:
cool thanks for the clarification. circleci changes are ready to :ship: